src/mission_execution/package.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>mission_execution</name>
+  <version>0.1.0</version>
+  <description>TODO</description>
+  <author email="oregon.robotics.team@intel.com">Oregon Robotics Team</author>
+  <maintainer email="oregon.robotics.team@intel.com">Oregon Robotics Team</maintainer>
```

Comment:
I think we should remove this email address from our commits till it actually exists. Otherwise, people may start trying to use it.

Reply:
Ok, I'll remove any stray author entries.